File: Python/1768_Merge_Strings.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        combined_word = ""
        same_length = min(len(word1), len(word2))
        for i in range(same_length):
            combined_word = combined_word + word1[i] + word2[i]

        if len(word1) > len(word2):
            for i in range(len(word1)):
                if i > same_length-1:
                    combined_word = combined_word + word1[i]
        if len(word2) > len(word1):
            for i in range(len(word2)):
                if i > same_length-1:
                    combined_word = combined_word + word2[i]
        
        return combined_word

def main():
    str1 = "ABCABC"
    str2 = "ABC"
    solution = Solution()
    answer = solution.mergeAlternately(str1, str2)
    print(answer)

if __name__ == "__main__":
    main()

File: Java/1768_Merge_Strings.java
package Java;
class MergeStrings {

    // Outputs a String that takes in String word1 and String word2.
    public String mergeAlternately(String word1, String word2) {
        System.out.println("word1: " + word1);
        System.out.println("word2: " + word2);
        return "";
    }

    public static void main(String[] args) {
        MergeStrings mergeStrings = new MergeStrings();
        String result = mergeStrings.mergeAlternately("abc", "pqr");
        System.out.println("Merged string: " + result);
    }
}